src/app/pages/IdxPage/index.stories.jsx

```diff
@@ -18,8 +17,8 @@ const stories = storiesOf('Pages|Idx Page', module).addDecorator(withKnobs);
     pageData: ukraineInRussianIdxData,
     service: 'ukrainian',
   },
-].forEach(({ idxPage, pageData, service }) => {
-  stories.add(`${idxPage}`, () => (
-    <IdxPageWithContext service={service} pageData={pageData} />
-  ));
+].forEach(({ idxPage, pageData, service = 'persian' }) => {
```

Comment:
We don't need to declare a default service in this file since its already defined here https://github.com/bbc/simorgh/blob/f9b3fcba91a8f486b121ffd79c3d26142a7573c7/src/app/pages/IdxPage/testHelpers/index.jsx#L8